styles.css

```diff
@@ -71,6 +122,35 @@ html {
   color: #344563;
 }
 
+li {
+  list-style-type: none;
+}
+
+.linktonic {
+  display: flex;
+  align-content: flex-start;
+  text-align: center;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  letter-spacing: 0.03em;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
```

Comment:
[OPTIONAL]

Maybe you could reduce the `border-radius` to the buttons to make them closer to the Figma design :wink: